index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+
+    <title>SimCom Documentation</title>
+  </head>
+  <body>
+    <!-- <nav class="sticky--nav">
+
+      
+      <h3 class="hero">SimCom</h3>
+      <ul class="doc--nav--list">
+        <li class="doc--nav--items"><a href="#home">home</a></li>
+        <li class="doc--nav--items"><a href="./pages/components.html">docs</a></li>
+        <li class="doc--nav--items"><a href="#connect">connect</a></li>
+      </ul>
+    </nav> -->
+
+    <nav class="sticky--nav">
+      <h3 class="nav--hero">SimCom</h3>
+      <ul class="nav--list">
+        <li class="nav--items"><a href="#home">home</a></li>
+        <li class="nav--items"><a href="/pages/components.html">docs</a></li>
+        <li class="nav--items"><a href="#connect">about</a></li>
+      </ul>
+    </nav>
+
+    <!-- header  section -->
+    <header id="home" class="doc--section home">
+      <h1>
```

Comment:
It would be great if you just add one '-' in class name i.e class="doc-section-home".
Just a personal suggestion though.